sum_item_list.py

```diff
@@ -0,0 +1,21 @@
+#function to sum up items in a list
+def sum_items(a):
```

Comment:
Hey @Janet-Namutebi You were required to use recursion but it so happens that you just used loops, Please read about recursion and re-implement this. Recursion will help you avoid having nested for conditions. For example if I change the program input to add a list in a list in a list as well i.e
```py
 l = [1, 2, [2, 3, [1, 5]]]
```
You will need to write another if statement and a for .... But with recursion, one solution will handle all the nested levels

Reply:
Alright, let me look at it tomorrow